### ISSUE TITLE
get all images URI or Path from request storage access in a particular folder

### ISSUE THREAD

**Issue by HirenJodhani** [anggrayudi/SimpleStorage#57]
dear sir, (please java)

please discribe a scenario of get content(Image or Video) uri or Accessable path in android 11 scope storage. 
or add a this type example for your simpleStorage Library in java .

i am running in andorid 11 and i have not found any solution for get content (uri or accessable path) from ACTION_OPEN_DOCUMENT_TREE Intent. 

please help. sir

**Comment by anggrayudi** [comment]
Once the user has granted the access to a specific folder, you can query all images with `search()` function:
```kotlin
DocumentFile grantedFolder = ...
List<DocumentFile> imageFiles = DocumentFileUtils.search(grantedFolder, true, DocumentFileType.FILE, new String[]{"image/*"});
```

**Comment by HirenJodhani** [comment]
dear sir,(Anggrayudi H)
Thank a lot This is My 5th day of get content in scope Storage. i get Solution for my Problem
Thanks Again.

Your Library is amazing and such helpful.

sir if you dont mind i can contact for hiring(In Future) my problem in android devlopment.

**Comment by anggrayudi** [comment]
Sure, thanks for your invitation. I'm glad this library helps you.
BTW, I've released version [`0.13.0`](https://github.com/anggrayudi/SimpleStorage/releases/tag/0.13.0).
